build-system/tasks/compile-bind-expr.js

```diff
@@ -32,7 +32,7 @@ gulp.task('compile-bind-expr', function() {
   const suppressCheckTypes = '/** @fileoverview ' +
       '@suppress {checkTypes, suspiciousCode, uselessCode} */';
   const imports = 'import {AstNode, AstNodeType} from \'./bind-expr-defines\';';
-  const jsExports = 'exports.parser = parser;';
+  const jsExports = 'export const publicParser = parser;';
```

Comment:
@choumx do you have a recommendation on what the exported name should be?

we can't use `parser` because the generated code already uses this as a var in the current module

Reply:
`bindParser`, `accessParser` etc.?